ISPN-6571 Adapt to configuration template changes

vblagoje opened the pull request:
Master only. @ryanemerson see ISPN-6571 for details. We need this one integrated before others. Please review thoroughly.

ryanemerson commented on src/main/webapp/components/directives/cache/local-cache.html:
```diff
@@ -34,7 +34,7 @@
       <configuration-section fields="[{name: 'Transactions', fields:['locking', 'mode', 'stop-timeout']}]" data="data.transaction.TRANSACTION"
                              metadata="resolveDescription('tx')" init-defaults="initDefaults" read-only="readOnly"/>
     </uib-tab>
-    
```
Should we avoid unnecessary style changes to improve diffs?

vblagoje replied:
Right @ryanemerson I have to turn off automatic on save formatter in Webstorm.

ryanemerson commented:
@vblagoje Overall it looks good, however you need to update the message in [confirmation-cache-modal.html](https://github.com/infinispan/infinispan-management-console/blob/master/src/main/webapp/edit-cache/confirmation-cache-modal.html).  It currently outputs "Update configuration template false" as it still references ```configurationModel.template```

vblagoje commented:
Awesome @ryanemerson Man I can't tell you how glad I am to have you on the team and that I am not in this alone!! I updated changes but HTML tab/blank line formatting is still an issue. I added a commit to PR so if you can fix it as you squash commits I would be grateful. I'll adjust Webstorm not to edit these in the future.

ryanemerson commented:
Integrated with 38d8cc5f892757ea89d921b77ab3b7de35eb41bc. Thanks @vblagoje
